routes.go

```diff
@@ -9,7 +9,10 @@ func alive(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintln(w, "Welcome to whatsforlunch!")
 }
 
-// Routes contains all routes for the alive status
+// TODO: Do this better.
+var locationHandlers = NewLocationHandlers()
+
+// GetAllRoutes contains all routes
 func GetAllRoutes() []*Route {

```

Comment:
This is a cool pattern 👍 . We haven't been listening routes like this in our code, but most of our svcs have very few endpoints.